riposte-core/src/main/java/com/nike/riposte/util/ErrorContractSerializerHelper.java

```diff
@@ -40,7 +40,11 @@ public static ErrorResponseBodySerializer asErrorResponseBodySerializer(ObjectMa
                     //      indicates empty response body payload, so we should return null.
                     return null;
                 }
-                return objectMapper.writeValueAsString(errorResponseBody.bodyToSerialize());
+                if(errorResponseBody.bodyToSerialize() instanceof CharSequence) {
```

Comment:
Pull `errorResponseBody.bodyToSerialize()` into a variable so we're not calling it multiple times. Although it would be unwise to do something expensive in that method, we can't control how people will use it, so better safe than sorry.

Reply:
And I guess that would need to be up top, before the null checks. Hmm. Maybe:
```
Object bodyToSerialize = (errorResponseBody == null) ? null : errorResponseBody.bodyToSerialize();
```
And then the null check would just become `if (bodyToSerialize == null) { ...`, and we can use `bodyToSerialize` later if it passes the null check.